samples/tests/mission1-local.yaml

```diff
@@ -0,0 +1,55 @@
+drone:
```

Comment:
i think i renamed this sample to merge local and docker samples. you can remove it if not needed in your code